src/plugins/controls/public/react_controls/control_group/components/control_group_editor.tsx

```diff
@@ -90,7 +90,10 @@ export const ControlGroupEditor = ({ onCancel, onSave, onDeleteAll, stateManager
       </EuiFlyoutHeader>
       <EuiFlyoutBody data-test-subj="control-group-settings-flyout">
         <EuiForm component="form" fullWidth>
-          <EuiFormRow label={ControlGroupStrings.management.labelPosition.getLabelPositionTitle()}>
+          <EuiFormRow
+            display="rowCompressed"
+            label={ControlGroupStrings.management.labelPosition.getLabelPositionTitle()}
+          >
```

Comment:
Why the change to the editors? I assumed the attached issue was only for the controls, not the editors - possible I missed something, though :eyes: